cli/jell/model/states.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+type State string
+
+const (
+	menu   State = "menu"
+	search State = "search"
+	github State = "github"
+)
```

Comment:
Måske et lidt mere beskrivende typenavn end "State"

Nitpick:
Du kan også lave det som en int, og bruge iota, så minder det lidt om en enum i andre sprog
```suggestion
type State int

const (
	menu   State = iota
	search State
	github State
)
```